### ISSUE TITLE
I get these errors

### ISSUE THREAD

**Issue by TheoryOfGabi** [OriginalAlien/Roblox-Mass-Report#9]
**So basically I get this error when I try to execute it:**

Traceback (most recent call last) :
File "D: (Muie\Roblox-Mass-Report-main\main.py", line 37, in <module>
  RB.report(victim, amount, reason, cooldown, reasonDescriptions[reason])
File "D: line 108, in report
  id = json. loads (requests .get(

KeyError : 'Id'

![Captură ecran (1106)](https://user-images.githubusercontent.com/122385102/235231254-77287765-5b37-4922-a77a-c35b2f6a90b3.png)

**Comment by OriginalAlien** [comment]
yeah roblox updated the api to getting a id from a roblox username, i updated it and added some extra features, lmk if theres anymore bugs/errors
